### ISSUE TITLE
Google Collab - Error “Could not load the JavaScript files needed to display output. (…)”

### ISSUE THREAD

**Issue by lacorreia65** [googlecolab/colabtools#2209]
I am having problems running a Python notebook in Google Collab.

Please find below detailed information

Bug report for Colab: http://colab.research.google.com/.

• Describe the current behavior: I've been using Collab for a while now and never had issues, but since yesterday it started showing an error. It says: Error Could not load the JavaScript files needed to display output. This is probably because your Google account login access has expired or because third-party cookies are not allowed by your browser. Please reload this page.

![image](https://user-images.githubusercontent.com/35398097/131230285-6cfbbc58-1df7-4de6-a830-6b752d8a5220.png)

I have checked and none of the two possible problems it says are true.. my google account access and google-collab have not expired and I have 3rd party cookie enabled. (it is disabled only in incognito mode).

I have tried a bunch of things, including some pointed [https://github.com/googlecolab/colabtools/issues/745/](url) and [https://stackoverflow.com/questions/56550349/google-colab-running-any-cell-received-javascript-related-errors](url)

Here are details of what I tried:

1- Restarted chrome, disabled/enables 3rd party cookies and login again… failed.

2- Restarted the machine… failed

3- Restarted the browser with chrome://restart… failed

4- Included domains https://colab.research.google.com and [*.]google.com in white-list allowing the use of 3rd-party cookies… failed

5- Tried to run the notebook under VPN… failed

6- Enabled/disabled GPU usage… failed

7- Included https://colab.research.google.com as javascript white-list of websites

8- Restarted the machine several times after each change… failed

9- Logged off/logged in my google account (… and restarted again).. failed

10- Deleted all Cookies, History Data (… plus additional restarts for each deletion)… failed.

11- Enabled “all cookies allowed”… failed.

12- Updated Javascript version, restarted the machine… failed.

13- Tried to run under incognito mode… failed. Nothing worked…

• Describe the expected behavior: It should execute the display of pandas dataset... it has done it several times before and suddenly stopped working.

• The web browser you are using (Chrome, Firefox, Safari, etc.): Google Chrome version Version 92.0.4515.159 (Official Build) (64-bit)

I have also checked the Developer Tools Console and saw the following error messages:

![Screenshot 08-28-2021 13 46 51](https://user-images.githubusercontent.com/35398097/131230365-2354d78c-5024-4902-ab96-1d427e1d15ea.png)

![Screenshot 08-28-2021 13 47 17](https://user-images.githubusercontent.com/35398097/131230371-df022387-ac27-4fae-9003-e373de82bcdd.png)

Finally, I tested the python notebook with Firefox and it worked perfectly. The weird think is that Chrome was working yesterday and suddenly stopped working.

Thank you in advance for any help to fix this problem.

**Comment by blois** [comment]
For the third-party cookie domains I believe it needs to include 'https://*colab.googleusercontent.com' since the origins will be something like 'https://edu42sots4-496ff2e9c6d22116-0-colab.googleusercontent.com'.

But not doing this typically only prevents browser ServiceWorkers from working- most outputs should continue to render. If the outputs are not loading at all then my only guess is that an extension is blocking them? Can you try an incognito window?

**Comment by lacorreia65** [comment]
Hello Blois, thanks for your comments. In fact I also tested in incognito mode with same results (failed - see step #13).

I just included a 3rd-party domain as you suggested in Chrome cookie configuration as "https://$colab.googleusercontent.com" (using the "*" it says its invalid) and it worked :) 

I will keep testing for a couple of days and let you know if the problem come up again (hopefully not!)

Thanks again! :)
